github/activity_events.go

```diff
@@ -31,8 +31,50 @@ func (e Event) String() string {
 // (PushEvent), a value of the corresponding struct type will be returned.
```

Comment:
> For recognized event types (PushEvent), a value of the corresponding struct type will be returned.

This change makes that comment out of date. There's no longer just the single recognized event type `PushEvent`.